src/ReactPixiFiber.js

```diff
@@ -142,12 +165,45 @@ const ReactPixiFiber = ReactFiberReconciler({
         );
         break;
       default:
+        if (type in INJECTED_TYPES) {
+          const injectedType = INJECTED_TYPES[type];
+          let customDisplayObject;
+          if (typeof injectedType === "function") {
```

Comment:
This either needs to be documented in README or removed. Basically it allows to create `CustomPIXIComponent` using shorthand form of only `customDisplayObject`.